python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_chaos_cleaning_strategy.py

```diff
@@ -90,31 +86,31 @@
 #
 # Low-rank polynomial chaos expansion
 # -----------------------------------
-# For any :math:`\boldsymbol{\alpha} \in \mathbb{N}^p`, let
-# :math:`\|\boldsymbol{\alpha}\|_0` be the rank of the multiindex, that is,
+# For any :math:`\vect{\alpha} \in \mathbb{N}^p`, let
+# :math:`\|\vect{\alpha}\|_0` be the rank of the multiindex, that is,
 # the number of nonzero components:
 #
 # .. math::
 #
-#   \|\boldsymbol{\alpha}\|_0 = \sum_{i = 1}^p \boldsymbol{1}_{\alpha_i > 0}
+#   \|\vect{\alpha}\|_0 = \sum_{i = 1}^p \vect{1}_{\alpha_i > 0}
 #
-# where :math:`\boldsymbol{1}` is the indicator function.
+# where :math:`\vect{1}` is the indicator function.
 # The multiindex set of maximum total degree :math:`d \in \mathbb{N}`
 # and maximum rank :math:`j \in \mathbb{N}` is ([blatman2009]_ page 74):
 #
 # .. math::
 #
-#   \mathcal{A}^{d,j} = \left\{ \boldsymbol{\alpha} \in \mathbb{N}^p
-#   \; | \; \|\boldsymbol{\alpha}\|_1 \leq d, \;
-#   \; \|\boldsymbol{\alpha}\|_0 \leq j\right\}.
+#   \mathcal{A}^{d,j} = \left\{ \vect{\alpha} \in \mathbb{N}^p
+#   \; | \; \|\vect{\alpha}\|_1 \leq d, \;
+#   \; \|\vect{\alpha}\|_0 \leq j\right\}.
 #
 # Therefore, the rank-`j` polynomial chaos expansion is:
 #
 # .. math::
 #
-#   h(\boldsymbol{\xi}) = \sum_{\boldsymbol{\alpha} \in
-#   \mathcal{A}^{d,j}} a_{\boldsymbol{\alpha}}
-#   \psi_{\boldsymbol{\alpha}}(\boldsymbol{\xi}) + \epsilon.
+#   \tilde{h}(\vect{z}) = \sum_{\vect{\alpha} \in
```

Comment:
widetilde is larger and cover the symbol better:
```suggestion
#   \widetilde{h}(\vect{z}) = \sum_{\vect{\alpha} \in
```

Reply:
ok